robottelo/test.py

```diff
@@ -148,11 +148,16 @@ def setUp(self):  # noqa
             self._docker_browser = DockerBrowser()
             self._docker_browser.start()
             self.browser = self._docker_browser.webdriver
+            self.addCleanup(self._docker_browser.stop)
         else:
             self.browser = browser()
+            self.addCleanup(self.browser.quit)
```

Comment:
If browser was not able to open and we get some exception for it. Then cleanup will be called which will fail as `None.quit` produce nothing. Are we going to have two exceptions in result? Or only last one?